### PR TITLE
[PassKit] Update bindings to Xcode 9.3 Beta 1

### PR DESCRIPTION
**Pull request by dalexsoto**
Diff files:
- https://github.com/xamarin/xamarin-macios/wiki/PassKit-iOS-xcode9.3-beta1
- https://github.com/xamarin/xamarin-macios/wiki/PassKit-watchOS-xcode9.3-beta1

### CODE REVIEW

File: src/passkit.cs [hunk 1]
```diff
@@ -984,8 +984,33 @@ interface PKLabeledValue
...
 	}
 
+	[Watch (4,3), iOS (11,3)]
+	[BaseType (typeof (NSObject))]
+	interface PKTransitPassProperties {
+
+		[Static]
+		[Export ("passPropertiesForPass:")]
+		[return: NullAllowed]
+		PKTransitPassProperties GetPassProperties (PKPass pass);
```
Comment (spouliot) [inline src/passkit.cs]: that _hint_ to a `[DisableDefaultCtor]`, the fact that there's no setter seems to make it unusable if `init` was used (successfully).

looks like we should have done the same on `PKSuicaPassProperties` :|

Comment (dalexsoto) [reply src/passkit.cs]: Ahh good catch!

Comment (dalexsoto) [reply src/passkit.cs]: Ahh good catch! 👍

**Comment by dalexsoto** [comment]
@spouliot done, thanks!

**Pull request merged**